### ISSUE TITLE
`negate_assign ` documentation typo

### ISSUE THREAD

**Issue by survived** [rust-bitcoin/rust-secp256k1#320]
Documentation to [negate_assign] function says that it "will return an error if the pk would be invalid", but it returns unit type `()`. I believe it's a typo - negation of "not at infinity" point cannot result into point at infinity, right?

**Comment by real-or-random** [comment]
Indeed. The upstream function never fails: https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1.h#L646

**Comment by tcharding** [comment]
This issue is stale, docs for `negate_assign` no longer mention error return.

We can close this one.
